config/config.yaml

```diff
@@ -0,0 +1,104 @@
+# Wallet address, path to the wallet must be set as cli parameter or environment variable
+wallet:
+  passphrase: "" # Passphrase to decrypt wallet.
+
+# Account address. If omitted default one will be used.
+address: NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP
+
+# Nodes configuration
+# This configuration makes the gateway use the first node (grpc://s01.neofs.devenv:8080)
+# while it's healthy. Otherwise, gateway uses the second node (grpc://s01.neofs.devenv:8080)
+# for 10% of requests and the third node (grpc://s03.neofs.devenv:8080) for 90% of requests.
+# Until nodes with the same priority level are healthy
+# nodes with other priority are not used.
+# The lower the value, the higher the priority.
+peers:
+  0:
+    address: node1.neofs:8080
+    priority: 1
+    weight: 1
+  1:
+    address: node2.neofs:8080
+    priority: 2
+    weight: 0.1
+  2:
+    address: node3.neofs:8080
+    priority: 2
+    weight: 0.9
+
+# Address to listen and TLS
+listen_address: 0.0.0.0:8084
+tls:
+  cert_file: /path/to/cert
+  key_file: /path/to/key
+
+logger:
+  level: debug
+  format: console
+  trace_level: panic
+  no_caller: false
+  no_disclaimer: true
+  sampling:
+    initial: 1000
+    thereafter: 1000
```

Comment:
I think the only thing we actually set in the logger now is level.

Reply:
The idea was to provide an example file with all available params, and a user will delete unused.

Reply:
We just don't support other options

Reply:
Hm, missed that PR with refactoring